cmd/bnsapi/main.go

```diff
@@ -90,11 +90,11 @@ func run(conf Configuration) error {
 	rt.Handle("/gov/proposals", &handlers.GovProposalsHandler{Bns: bnscli})
 	rt.Handle("/gov/votes", &handlers.GovVotesHandler{Bns: bnscli})
 	rt.Handle("/gconf/", &handlers.GconfHandler{Bns: bnscli, Confs: gconfConfigurations})
-	rt.Handle("/", &handlers.DefaultHandler{Domain: conf.Domain})
+	rt.Handle("/", &handlers.DefaultHandler{Domain: conf.HostPort})
 
 	docs.SwaggerInfo.Version = util.BuildVersion
-	docs.SwaggerInfo.Host = conf.Domain
-	docsUrl := fmt.Sprintf("http://%s%s/docs/doc.json", conf.Domain, conf.HTTP)
+	docs.SwaggerInfo.Host = conf.HostPort
+	docsUrl := fmt.Sprintf("http://%s%s/docs/doc.json", conf.HostPort, conf.HTTP)
```

Comment:
Using `HOST_PORT` and `HTTP` breaks docUrl.  Maybe we need to re-think this since we're specifying the port in two different env vars.  I'll have to get back to you after my meetings this afternoon.

Reply:
:+1: